CMakeLists.txt

```diff
@@ -215,7 +215,7 @@ if(NOT DEACTIVATE_IPP)
 endif()
 
 if(BUILD_PLUGINS)
-    set(HAVE_PLUGINS TRUE)
```

Comment:
In my local c-blosc2 this makes the following config.h:
![scrsh](https://user-images.githubusercontent.com/45040313/122740249-13860b80-d284-11eb-8553-b9d007c781ca.png)
Then HAVE_PLUGINS appears as undefined and blosc_init() runs the lines
  register_codecs();
  register_filters();
but it should not.

Reply:
Hmm. I wonder why it wasn't registering for me. I will have to take a look at it.

Reply:
From the Blosc Development Team, accordingly to our last meeting we agreed that the use of the set() command should be kept for consistency reasons.

Reply:
I didn't mean to cause a meeting. I wasn't aware of the config so you were right of course. I think perhaps my config header was not being read/written because I am building from a different directory.. I have added a commit which should fix that.